cookbook/configuration/mongodb_session_storage.rst

```diff
@@ -0,0 +1,54 @@
+How to Use MongoDbSessionHandler to Store Sessions in a MongoDB Database
+========================================================================
+
+The default Symfony session storage writes the session information to files.
+Some medium to large websites use a NoSQL database called MongoDB to store the
+session values instead of files, because databases are easier to use and scale
+in a multi-webserver environment.
+
+Symfony has a built-in solution for NoSQL database session storage called
+:class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\MongoDbSessionHandler`.
+MongoDB is an open-source document database that provides high performance,
+high availability, and automatic scaling. This article assumes that you have
+already `installed and configured a MongoDB server`_. To use it, you just
+need to change/add some parameters in the main configuration file:
+
+.. configuration-block::
+
+    .. code-block:: yaml
```

Comment:
Good point, however, I do think it would add clarity to show how this is done, so I moved the parameter definitions to a separate section of the article in my latest commit.  @WouterJ would you please review the XML and PHP sections of the new addition?